scripts/create-merged-schema.ts

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env ts-node
+/*
+ * Create a complete JSON schema for tsconfig.json
+ * by merging the schemastore schema with our ts-node additions.
+ * This merged schema can be submitted in a pull request to
+ * SchemaStore.
+ */
+
+import axios from 'axios';
+import * as Path from 'path';
```

Comment:
Nit: Casing on `Path` should be `path`, but you could just import `resolve` only.